src/case-studies/yubico/YubicoCaseStudy.module.css

```
@@ -81,89 +76,218 @@
+}
+
+.overlayCloseText {
+  line-height: 1;
+}
+
+.visuallyHidden {
+  position: absolute;
+  overflow: hidden;
+  width: 1px;
+  height: 1px;
+  clip: rect(0 0 0 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)